src/Sulu/Bundle/AdminBundle/Resources/js/services/Translator/Translator.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type {TranslationMap} from './types';
+
+let translationMap: ?TranslationMap;
+
+function setTranslations(translations: TranslationMap) {
+    translationMap = translations;
+}
+
+function clearTranslations() {
+    translationMap = null;
+}
+
+function translate(key: string) {
+    if (!translationMap || !(key in translationMap)) {
+        throw new Error('Translation for key "' + key + '" not found');
```

Comment:
Maybe returning the key instead of throwing an error would be developer friendly?